src/windows/gfx/d3d11-ctx.c

```diff
@@ -9,7 +9,7 @@ GFX_CTX_PROTOTYPES(_d3d11_)
 
 #define COBJMACROS
 #include <d3d11.h>
-#include <dxgi1_3.h>
+#include <dxgi1_6.h>
```

Comment:
Good point. Yes, HDR is best supported from Windows 10 onwards. Now, whether we are OK to completely remove Win 8.1 support for matoya is a question above my paygrade. 

If we do want to support 8.1 for non-HDR scenarios, I'd need to query windows version and conditionally do the HDR logic.

Reply:
After discussion, it was decided that the best thing to do is to GetModuleHandle + GetProcAddress the required HDR-specific functions and handle them if they are not present. This will allow for backwards compatibility.

Reply:
While we are deprecating Windows 8.1 for Parsec, libmatoya is another matter